hack/label.sh

```diff
@@ -14,7 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Update release labels
-export TAG=${TAG:-$(git rev-parse HEAD)}
-echo "Updating release labels to app.kubernetes.io/version: \"${TAG}\""
-export LABEL_YAML_CMD=(sed -e "s|app.kubernetes.io/version: devel|app.kubernetes.io/version: \"${TAG}\"|")
```

Comment:
This was the actual issue. This file was sourced before TAG (/its dependent fields) was set correctly in [parse_flags](https://github.com/knative-extensions/eventing-kafka-broker/blob/7b37a956abed40085408347ae01346e34dbf5127/vendor/knative.dev/hack/release.sh#L442).